src/components/Content/Editor/TabMenu.svelte

```diff
@@ -47,9 +47,8 @@
     </ContextMenuOption>
     <ContextMenuOption labelText="Show in Explorer" on:click={() => {
         let dir = filepath.split(path.sep);
-        dir.pop();
-        dir = dir.join(path.sep);
-        shell.open(dir);
+        let explorerPath = dir.join(path.sep);
```

Comment:
There's no need to make a new variable here. Since `dir` is just `filepath` separated, joining it back together is repetitive. Might as well just use `filepath`.

Reply:
I have made the necessary changes